### PR TITLE
[RFC][NI] Fix tests failing

### PR DESCRIPTION
**Pull request by AndreJoaquim**
NI

### CODE REVIEW

File: tests/unit/utils/countries-test.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ test('getCountryName returns null and logs error on empty string', function(asse
   assert.expect(2);
 
   let oldError = Logger.error;
-  let expectedErrorMessage = 'The country with code  was not found'
+  let expectedErrorMessage = 'The country with code  was not found';
```
Comment (telmaantunes) [inline tests/unit/utils/countries-test.js]: remove extra space in "code  was". :rofl: jk, approved

Comment (AndreJoaquim) [inline tests/unit/utils/countries-test.js]: :rofl:

Comment (AndreJoaquim) [inline tests/unit/utils/countries-test.js]: I will improve it when I have some free time

Comment (telmaantunes) [inline tests/unit/utils/countries-test.js]: :1st_place_medal:

**Pull request merged**